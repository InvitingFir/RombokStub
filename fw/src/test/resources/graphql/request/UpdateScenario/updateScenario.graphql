mutation updateScenario($uuid: String!, $updates: ScenarioRequest!){
  updateScenario(scenarioUuid: $uuid, updates: $updates){
    scenarioType
    name
    uuid
    isDefault
    variables {
      name
      value
    }
    url
  }
}