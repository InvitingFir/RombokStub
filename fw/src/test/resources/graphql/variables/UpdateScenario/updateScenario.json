{
  "updates": {
    "scenarioType": "HTTP",
    "action": "updatedAction",
    "isDefault": false,
    "name": "updatedName",
    "predicate": "updatedPredicate",
    "url": "updatedUrl",
    "variables": {
      "name": "updatedName",
      "value": "updatedValue"
    }
  },
  "uuid": "9161f68d-da9e-4fa2-b0ad-142bce3b4550"
}