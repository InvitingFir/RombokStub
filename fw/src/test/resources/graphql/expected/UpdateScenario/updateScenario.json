{
  "data": {
    "updateScenario": {
      "scenarioType": "HTTP",
      "name": "updatedName",
      "uuid": "5ce891dc-b61e-4e8a-b183-21b58c31ba66",
      "isDefault": false,
      "variables": [
        {
          "name": "updatedName",
          "value": "updatedValue"
        }
      ],
      "url": "updatedUrl"
    }
  }
}